packages/components/src/input-control/types.ts

```diff
@@ -105,7 +122,18 @@ export interface InputFieldProps extends BaseProps {
 	paddingInlineStart?: CSSProperties[ 'paddingInlineStart' ];
 	paddingInlineEnd?: CSSProperties[ 'paddingInlineEnd' ];
 	setIsFocused: ( isFocused: boolean ) => void;
+	/**
+	 * Whether to wrap input field in a Tooltip component.
+	 *
+	 * @default false
+	 */
+	showTooltip?: boolean;
 	stateReducer?: StateReducer;
+	/**
+	 * Custom text to display when tooltips are enabled. If not provided enabled
+	 * tooltips will fallback to using the label value.
+	 */
+	tooltipText?: string;
```

Comment:
Do you think we'll ever be in a situation in which we'd like to pass a `ReactNode` as the tooltip's content, rather than just a string? Or do you think it's actually a good idea to only accept a simple string of text in this scenario?

Reply:
Nevermind, it actually looks like `Tooltip` only support text anyway.